### ISSUE TITLE
Can't run

### ISSUE THREAD

**Issue by dragGH102** [boxxa/meteor-polymer-paper#4]
meteor add boxxa:polymer-paper
meteor run

=> Started proxy.  
=> Started MongoDB.  
I20150919-17:52:12.485(2)? installing polymer into public directory...
I20150919-17:52:20.768(2)? Polymer exist already. To reinitialize, remove bower_components from your public/ folder
I20150919-17:52:21.315(2)? installing polymer-paper-elements
W20150919-17:52:27.707(2)? (STDERR) 
W20150919-17:52:27.707(2)? (STDERR) /home/manuel/.meteor/packages/meteor-tool/.1.1.4.xvre8c++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:278
W20150919-17:52:27.707(2)? (STDERR)                                             throw(ex);
W20150919-17:52:27.707(2)? (STDERR)                                                   ^
W20150919-17:52:27.729(2)? (STDERR) polymer: Unable to find suitable version for polymer

**Comment by jmetz** [comment]
+1

**Comment by boxxa** [comment]
I'm gonna mark this as unsupported. I marked the polymer package for 1.0 as not supported as I moved away from Polymer.

**Comment by dragGH102** [comment]
@boxxa thank you for your answer.
But I didn't get the point. What package support what version of Meteor and Polymer?

**Comment by boxxa** [comment]
The error is bower related when it installs the library. It worked for the first 1.0 release and seems something changed. I can try to take a look today but I haven't kept up with it. Didn't realize this package was getting attention in atmosphere.

**Comment by dragGH102** [comment]
Yes, it would  be very handy. Polymer is a great library and it would be grate to use it in Meteor

**Comment by jmetz** [comment]
@dragGH102 - in the meantime, [meteor-elements](http://atoy40.github.io/meteor-elements/) seems to be working ok - I just cloned the example from [here](https://github.com/atoy40/meteor-polymer-music) and started playing around with it - provides components to handle collections and subscriptions (ddp).

**Comment by dragGH102** [comment]
thank you @Jeremy. I will try it out. I just wonder if it will work with
PolymerElements and not just customized elements, so I need to try.

**Comment by dandv** [comment]
> I'm gonna mark this as unsupported. I marked the polymer package for 1.0 as not supported as I moved away from Polymer.

Best to [hide it from Atmosphere then](https://github.com/percolatestudio/atmosphere/issues/267#issuecomment-58769824)? Esp. given that since v1.3, Meteor supports using [npm packages](https://www.npmjs.com/~polymer) directly.
